### PR TITLE
Crates blog post

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: _drafts/crates_meet_automation.rs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+layout: post
+author: gkoz
+title: Crates meet automation
+categories: [front, crates]
+---
+
+What's new with the latest batch of crates and the reason for some
+incompatible changes? More automation.
+
+
```
Comment (GuillaumeGomez) [inline _drafts/crates_meet_automation.rs]: I think this empty line is useless (it's not supposed to make sense to have 2 empty lines in markdown as far as I know).

File: _drafts/crates_meet_automation.rs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+### No more features
+
+You don't have to set the target library versions in your `Cargo.toml`.
+The build script will query the installed library versions from `pkg-config`
+and instruct `rustc` via `cfg` arguments to compile the appropriate set of APIs.
+
+All the APIs availiable in the installed library will just work but if you
```
Comment (GuillaumeGomez) [inline _drafts/crates_meet_automation.rs]: "availiable"

File: _drafts/crates_meet_automation.rs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Note that [a cargo issue][cargo-issue] will prevent `cargo doc` from generating the
+documentation or at least cause it to be incomplete.
+
+#### Migration
+
+Remove any `features` and `default-features` directives relating to Gtk-rs
+crates from your `Cargo.toml`.
+
+
```
Comment (GuillaumeGomez) [inline _drafts/crates_meet_automation.rs]: Same here.

File: _drafts/crates_meet_automation.rs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+### Introducing autogenerated sys crates
+
+From now on the FFI definitions crates (except `cairo-sys`) don't have to
```
Comment (GuillaumeGomez) [inline _drafts/crates_meet_automation.rs]: "From now on the FFI definitions"... I thin a comma after "From now on" would make the reading more pleasant (just an opinion however).

File: _drafts/crates_meet_automation.rs (hunk 1 above)
Comment (gkoz) [inline _drafts/crates_meet_automation.rs]: It shouldn't have any effect on output but helps a bit in the raw form.

Comment (GuillaumeGomez) [inline _drafts/crates_meet_automation.rs]: That's what I thought, but it's less "clean". ;)

**Pull request merged**